lib/foreman_maintain/customize_core.rb

```diff
@@ -0,0 +1,6 @@
+class String
```

Comment:
I've seen multiple gems having `core_ext.rb` file for this kind of things. We can follow this here as well